docs/getting-started.md

```diff
@@ -52,10 +52,12 @@ Remove nodes
 You may want to remove **worker** nodes to your existing cluster. This can be done by re-running the `remove-node.yml` playbook. First, all nodes will be drained, then stop some kubernetes services and delete some certificates, and finally execute the kubectl command to delete these nodes. This can be combined with the add node function, This is generally helpful when doing something like autoscaling your clusters. Of course if a node is not working, you can remove the node and install it again.
 
 - Add worker nodes to the list under kube-node if you want to delete them (or utilize a [dynamic inventory](https://docs.ansible.com/ansible/intro_dynamic_inventory.html)).
```

Comment:
delete  it.

Reply:
What the complete chapter?

Reply:
Because we delete the change of node mode, we need to delete 54 lines.